index.html

```diff
@@ -54,6 +51,12 @@ <h1 class="summary-title"></h1>
             </article>
         </template>
 
+        <template id="blog_image_template">
+            <div class="blog-image-container">
```

Comment:
Is this container div strictly necessary? Seems like the IMG tag would be sufficient on its own?